### PR TITLE
No refresh on pantry list and recipe! 

### PR DESCRIPTION
**Pull request by samforstot**
## Describe your changes
Now the page no longer refreshes when adding items to pantry, list, or scrolling through recipes
## Issue ticket number and link
#434

### CODE REVIEW

File: src/Scrapers/KrogerScraper.py
Comment (rileycleavenger) [inline src/Scrapers/KrogerScraper.py:1]: idk how this made it in there, it isn't even the finished version, can you remove it from this PR?

**Review by rileycleavenger — CHANGES_REQUESTED** [review]
Ok this change is BIG TIME, it makes the app run so much more seamlessly, its mad hype. However, here are some things I thought of that you should add before we merge this:

- [x] Change the "Add Cookbook" button to also not require a refresh (similar to save filters button)
- [x] When selecting "Add Cookbook" or "Save Filters" hide their respective menus so that users know their selection worked
- [ ] Verify that when adding an item to your pantry it will update the recommended recipes accordingly without the need for a refresh. Based on backend it should, but you should confirm this.

File: src/Scrapers/KrogerScraper.py
Comment (samforstot) [reply src/Scrapers/KrogerScraper.py:1]: Yeah I gotchu, couldve been a git add . from before

**Comment by samforstot** [comment]
@rileycleavenger All of these things are now solved, found a new bug where when the pantry is empty it requires a refresh

**Comment by samforstot** [comment]
Solved previous bug, will add no refresh on deals page soon.

**Comment by rileycleavenger** [comment]
Fire, will review after you change the deals page

File: Saucier720-app/src/app/recipes/FEC/filter-menu/filter-menu.component.ts
Comment (rileycleavenger) [inline Saucier720-app/src/app/recipes/FEC/filter-menu/filter-menu.component.ts:64]: can you remove all of the commented out refreshes?

File: Saucier720-app/src/app/recipes/FEC/new-recipe/new-recipe.component.ts
Comment (rileycleavenger) [inline Saucier720-app/src/app/recipes/FEC/new-recipe/new-recipe.component.ts:78]: here

File: Saucier720-app/src/app/recipes/FEC/recipe-nav-bar/recipe-nav-bar.component.ts
Comment (rileycleavenger) [inline Saucier720-app/src/app/recipes/FEC/recipe-nav-bar/recipe-nav-bar.component.ts:69]: here

File: Saucier720-app/src/app/recipes/FEC/recipe-nav-bar/recipe-nav-bar.component.ts
Comment (rileycleavenger) [inline Saucier720-app/src/app/recipes/FEC/recipe-nav-bar/recipe-nav-bar.component.ts:91]: here

**Review by rileycleavenger — CHANGES_REQUESTED** [review]
I know this is nit picky, but can you remove the commented out refreshes? Also will approve after making deals page static

**Review by rileycleavenger — APPROVED** [review]
LGTM! These changes are game changer for running the app as an SPA, it makes it so much more fluid. I love.

**Pull request merged**
